src/GameStateMenu.cpp

```diff
@@ -10,143 +10,509 @@
+                    }
+        		}
+        	}
+
+        	break;
+        case SDL_WINDOWEVENT:
+        	switch (event.window.event) {
+	    		case SDL_WINDOWEVENT_RESIZED:
+                    //Adjust the dimensions of the window if resized
+	    			level.levelTexture.setDimensions(game.window.getWidth(),
+                                                    game.window.getHeight());
+	    			break;
+	        }
+        	break;
+		case SDL_QUIT:
+			play = false;
+			break;
+      	default:
+        	break;
+	}
 }
 
+/**
+* Function: update
+*
+* Date:
+* JF: February 8, 2017: Moved positional data into own method
+*
+* Designer:
+* Jacob Frank
+*
+* Programmer:
+* Jacob Frank
+*
+* Interface: update(const float& delta)
+*
+* Returns: void
+*
+* Notes:
+* Function positions all screen elements in the window
+*/
 void GameStateMenu::update(const float& delta) {
 
-    // TEMP: Skip to GameStateMatch
-    // Remove this when working on the main menu
-    game.stateID = 2;
-    play = false;
-
+	if(delta == JOIN) {
+		game.stateID = 2;
+	} else if (delta == OPTIONS) {
+        game.stateID = 2; //TEMPORARY: change to correct state ID once implemented
+	} else {
+        game.stateID = 0;
+	}
 }
 
-void GameStateMenu::render() {
-    //Only draw when not minimized
-    if ( !game.window.isMinimized() ) {
-
-        //Clear screen
-        SDL_SetRenderDrawColor( game.renderer, 0xFF, 0xFF, 0xFF, 0xFF );
-        SDL_RenderClear( game.renderer );
-
-        SDL_Color textColor = { 0, 0, 0, 255 };
+/**
+* Function: positionElements
+*
+* Date:
+* JF: March 6, 2017: Moved positional data into own method
+*
+* Designer:
+* Jacob Frank
+*
+* Programmer:
+* Jacob Frank
+*
+* Interface: positionElements()
+*
+* Returns: void
+*
+* Notes:
+* Function positions all screen elements in the window
+*/
+void GameStateMenu::positionElements() {
+
+    int         windowWidth     = game.window.getWidth();
+    int         windowHeight    = game.window.getHeight();
+    int         maxTextWidth    = 0;
+    int         maxTextHeight   = 0;
+    int         vertPadding     = 50;
+    int         horzPadding     = 10;
+    std::string longestString   = "";
+    char        largestChar     = 'W';
+
+
+    //Check if TTF was initialized correctly
+    if(!TTF_WasInit() && TTF_Init()==-1) {
+        printf("TTF_Init: %s\n", TTF_GetError());
+        exit(1);
+    }
+    //Calculate the pixel length and height of the largest possible string
+    longestString.resize(maxLength+1, largestChar);
+	TTF_SizeText(textboxFont, longestString.c_str(), &maxTextWidth, &maxTextHeight);
+
+    //Position the menu text
+    menuItemPos[JOIN].x     = windowWidth/2  - menuTextTextures[JOIN].getWidth()/2;
+    menuItemPos[JOIN].y     = windowHeight/2 + menuTextTextures[JOIN].getHeight();
+    menuItemPos[OPTIONS].x  = windowWidth/2  - menuTextTextures[OPTIONS].getWidth()/2;
+    menuItemPos[OPTIONS].y  = windowHeight/2 + menuTextTextures[OPTIONS].getHeight()*2.2;
```

Comment:
in general try and space out operands and operators with a space between each ie 
```= windowWidth/2  - menuTextTextures[JOIN].getWidth()/2;```
is
```= windowWidth / 2  - menuTextTextures[JOIN].getWidth() / 2;```